app/errors.js

```diff
@@ -8,3 +8,9 @@ exports.databaseError = message => internalError(message, exports.DATABASE_ERROR
 
 exports.DEFAULT_ERROR = 'default_error';
 exports.defaultError = message => internalError(message, exports.DEFAULT_ERROR);
+
+exports.EXIST_USER = 'exist_user';
+exports.existUser = () => internalError('User already exist', exports.EXIST_USER);
```

Comment:
the message should be received as a parameter and could be a constant or send from service/controller